newrelic/resource_helpers_synthetics.go

```diff
@@ -267,30 +267,28 @@ func listValidSyntheticsMonitorStatuses() []string {
-
-		// "MUTED" to be removed from the provider as this is soon going to reach its EOL, on February 29, 2024
-		string(synthetics.SyntheticsMonitorStatusTypes.MUTED),
+		// synthetics.SyntheticsMonitorStatusTypes.MUTED removed on February 29, 2024 in accordance with its EOL
 	}
 }
 
 // validate function that validates the status of Synthetic Monitors
-// recent addition: return a warning if 'MUTED' status is used, as this has been deprecated
+// recent addition: return an error if 'MUTED' status is used, as this has reached EOL
 func validateSyntheticMonitorStatus(val interface{}, key string) (warns []string, errs []error) {
 	monitorStatusInput := val.(string)
 	listOfValidSyntheticMonitorStatuses := listValidSyntheticsMonitorStatuses()
 	containsValidSyntheticMonitorStatus := slices.Contains(listOfValidSyntheticMonitorStatuses, monitorStatusInput)
 	if !containsValidSyntheticMonitorStatus {
-		errs = append(errs, fmt.Errorf("expected status to be one of %v, got %s", listOfValidSyntheticMonitorStatuses, monitorStatusInput))
-	}
-
-	// hard-coding "MUTED" instead of using synthetics.SyntheticsMonitorStatusTypes.MUTED as it could be removed from newrelic-client-go post the EOL
-	if monitorStatusInput == "MUTED" {
-		warns = append(warns, `
-The 'MUTED' status of Synthetic Monitors has been deprecated, and shall reach its end of life in February 2024.
-In accordance with this, the New Relic Terraform Provider would also discontinue support for the status 'MUTED' soon, in an upcoming release.
-To mute Synthetic Monitors, please shift to alternatives such as muting rules. 
-A detailed guide on this can be found here: https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/guides/upcoming_synthetics_muted_status_eol_guide
-`)
+		// hard-coding "MUTED" instead of using synthetics.SyntheticsMonitorStatusTypes.MUTED as this has been removed from newrelic-client-go, owing to the EOL
+		if strings.ToUpper(monitorStatusInput) == "MUTED" {
+			errs = append(errs, fmt.Errorf("invalid monitor status 'MUTED' \n"+
+				"As of February 29, 2024, Synthetic Monitors no longer support the `MUTED` status.\n"+
+				"Version 3.33.0 of the New Relic Terraform Provider is released to coincide with the `MUTED` status end-of-life.\n"+
+				"Consequently, the only valid values for `status` for all types of Synthetic Monitors are `ENABLED` and `DISABLED`.\n"+
+				"If you have a Terraform configuration with Synthetic Monitor resources previously applied with the status `MUTED` and are hence, seeing this error now upon `terraform plan`,\n"+"please change the status of the monitor to one of the two aforementioned values to plan and apply your configuration, and opt for other alternatives to mute monitors.\n"+
+				"For additional information on alternatives to the `MUTED` status of Synthetic Monitors that can be managed via Terraform,\n"+"please refer to the Synthetic Monitors MUTED Status EOL Guide in the documentation of the New Relic Terraform Provider.\n"))
```

Comment:
Thoughts on providing the URL to link to this EOL guide? It would make it easier for the customer to get to it versus them having to go search for it in our docs.

Reply:
Yeah - this is something I _did_ consider but was wondering if that would make the already long error message even longer :) maybe I'll add it, I think that shouldn't be a problem

Reply:
Added the link to the error message. Thank you for the find :)